bears/python/PyDocStyleBear.py

```diff
@@ -1,15 +1,40 @@
 from coalib.bearlib.abstractions.Lint import Lint
 from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
 
 
 class PyDocStyleBear(LocalBear, Lint):
     executable = 'pydocstyle'
-    arguments = '{filename}'
     output_regex = r'(.*\.py):(?P<line>\d+) (.+):\n\s+(?P<message>.*)'
     use_stderr = True
 
-    def run(self, filename, file):
+    def run(self,
+            filename,
+            file,
+            pydocstyle_cli_options: str="",
+            pydocstyle_select: typed_list(str)=[],
+            pydocstyle_ignore: typed_list(str)=[]):
         '''
         Checks python docstrings.
+
+        :param pydocstyle_cli_options: Any command line options you wish to
+                                       be passed to PyDocStyle.
+        :param pydocstyle_select:      List of checked errors by specifying
+                                       which errors to check for.
+        :param pydocstyle_ignore:      List of checked errors by specifying
+                                       which errors to ignore.
         '''
+        self.arguments = '{filename}'
+        if pydocstyle_cli_options:
+            self.arguments += " " + pydocstyle_cli_options
+        if pydocstyle_ignore and pydocstyle_select:
+            self.err(
+                'Cannot pass both pydocstyle_ignore and pydocstyle_select.'
+                ' They are mutually exclusive. Neither will be applied.')
```

Comment:
@sils1297 
> you should put a return here right?

No. The execution will continue without pydocstyle_ignore and pydocstyle_select settings applied.